app/posts/apply-for-teacher-training/2019-12-18-digital-accessibility-centre-report.md

```diff
@@ -0,0 +1,110 @@
+
+This was [fixed by changing the error message](https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/1012) to "Enter feedback for the candidate". ([Trello](https://trello.com/c/7JcO0QxH/695-dac-page-20-add-error-text-for-blank-application-reject-reason))
+
+Automated accessibility testing would not have found this issue.
+
+### Reflow (WCAG AA)
+
+Page 23: On mobile devices links in the support page header were hidden and were not accessible.
+
+This has been fixed by showing a menu link which toggles hiding and showing of the links. It was [probably fixed when the header component was refactored](https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/982).
+
+### Non-descriptive form elements (WCAG AA)
+
+Page 27: A button to manage which candidates are used for reporting did not have an adequate description.
+
+> The ‘Hide in reporting’ and ‘Show in reporting’ buttons are not descriptive enough for all users, particularly screen reader users, to determine their function or purpose.
+
+This was [fixed by expanding the description of the button](https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/1025). ([Trello](https://trello.com/c/LIxkhgmi/696-dac-page-27-add-context-to-show-hide-in-reporting-button-in-support-application-page))
+
+Automated accessibility testing would not have found this issue.
+
+### Status message (WCAG AA)
+
+Page 29: When a success message was shown following an action the message was not focused, meaning screenreader users would not be aware of it.
+
+This was fixed [by making it focusable](https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/963) and [by adding an aria alert role to the element](https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/969). ([Trello](https://trello.com/c/83AycElp/683-dac-fix-success-message-focus))
+
+The fix brings our success alert inline with the error summary component provided by the design system.
+
+Automated accessibility testing would not have found this issue.
+
+## WCAG AAA issues
+
+We also fixed the following issues:
+
+* [Page 33: Add hidden content to "Change" links on Work History review (and other) page(s)](https://trello.com/c/XsqlA0hu/698-dac-page-33-add-hidden-content-to-change-links-on-work-history-review-and-other-pages)
+* [Page 41: Change headings on provider offer page so they are in a hierarchical order](https://trello.com/c/n0wiBbfB/699-dac-page-41-change-headings-on-provider-offer-page-so-they-are-in-a-hierarchical-order)
+* [Page 50: Add (optional) to standard conditions in provider applications offer page](https://trello.com/c/hW5m9EFg/702-dac-page-50-add-optional-to-standard-conditions-in-provider-applications-offer-page)
+* [Page 51: Change all pages to only ask for one thing per page](https://trello.com/c/QJVmXusT/703-dac-page-51-change-all-pages-to-only-ask-for-one-thing-per-page)
```

Comment:
Is this an AAA requirement?

Reply:
You're right, these are usability issues, I'll make a new section.